tests/rules/no-store-subscription.test.ts

```diff
@@ -11,17 +11,45 @@ ruleTester().run(path.parse(__filename).name, rule, {
       @Component()
```

Comment:
Do you know how could I test a single file with `uvu`? Also, is there any way to check what test file is not behaving as expected? Currently I got this:

<img width="693" alt="Captura de Tela 2021-08-30 às 10 57 54" src="https://user-images.githubusercontent.com/11965907/131350877-3060a6ff-1d32-4282-bd79-330943fabe47.png">

Reply:
Yikes, a single test can be run with the following command.
Sadly it doesn't support a glob.

```
node -r ts-node/register ./tests/rules/no-store-subscription.test.ts
```

For the AssertionError, I think we'll have to wrap the testrunner to also print the file 🤔
I'll try to work on this soon.